src/blurredLocationDisplay.js

```diff
@@ -7,11 +7,35 @@ BlurredLocationDisplay = function BlurredLocationDisplay(options) {
 
   options.Interface = options.Interface || require('./ui/Interface.js');
   options.blurredLocation = options.blurredLocation || {};
+  options.PLpeopleAPI = options.PLpeopleAPI || false ;
+  options.locations = options.locations || [] ;
+  options.source_url = options.source_url || "" ;
```

Comment:
We should clearly document what happens if you pass in locations AND pass a source URL.